app/views/notes/show.html.erb

```diff
@@ -130,3 +141,51 @@
   </div>
 
 </div><!--/span-->
+
+
+<style type="text/css">
+  .hexbin-hexagon {
+    stroke: #000;
```

Comment:
This seems to show both the name of the location and the map. But @ebarry may be able to comment on when we might show the textual location vs. a map location?